sql/core/src/main/scala/org/apache/spark/sql/parquet/CatalystSchemaConverter.scala

```diff
@@ -383,20 +386,14 @@ private[parquet] class CatalystSchemaConverter(
           .length(minBytesForPrecision(precision))
           .named(field.name)
 
-      case dec @ DecimalType() if !followParquetFormatSpec =>
-        throw new AnalysisException(
-          s"Data type $dec is not supported. " +
-            s"When ${SQLConf.PARQUET_FOLLOW_PARQUET_FORMAT_SPEC.key} is set to false," +
-            "decimal precision and scale must be specified, " +
-            "and precision must be less than or equal to 18.")
-
```

Comment:
You said it should usevthe hive default of (10,0) - or did I misinterpret that?

Am 28. Juni 2015 10:53:00 MESZ, schrieb Cheng Lian <notifications@github.com>:
>> @@ -383,20 +386,14 @@ private[parquet] class CatalystSchemaConverter(
>>            .length(minBytesForPrecision(precision))
>>            .named(field.name)
>>  
>> -      case dec @ DecimalType() if !followParquetFormatSpec =>
>> -        throw new AnalysisException(
>> -          s"Data type $dec is not supported. " +
>> -            s"When ${SQLConf.PARQUET_FOLLOW_PARQUET_FORMAT_SPEC.key}
>is set to false," +
>> -            "decimal precision and scale must be specified, " +
>> -            "and precision must be less than or equal to 18.")
>> -
>
>We still need this branch to handle the case where precision
>information is missing.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/apache/spark/pull/6796/files#r33420570

-- 
Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.